Web/starter_project_iconics/components/common/blogCard.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { ReactElement } from 'react'
+import styles from '../../styles/blogCard.module.css'
+import { Blog } from '../../types/models'
+import { MdOutlineModeComment } from 'react-icons/md'
+
+import {
+  AiFillClockCircle,
+  AiFillCheckCircle,
+  AiFillWarning,
+} from 'react-icons/ai'
+
+interface Props {
+  card: Blog
+  feature: string
+}
+
+const BlogCard: React.FC<Props> = ({ card, feature }) => {
+  const handleReadMore = () => {
+    console.log('Read More button clicked')
```

Comment:
better not to have console log in a mergeable code

Reply:
I've changed it now